taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/controller/Mapping.java

```diff
@@ -0,0 +1,13 @@
+package pro.taskana.adapter.camunda.outbox.rest.controller;
+
+/** Collection of Url to Controller mappings. */
+public class Mapping {
+
+  public static final String URL_EVENTS = "/events";
+  public static final String URL_EVENT = "/{eventId}";
+  public static final String URL_DELETE_EVENTS = "/delete";
```

Comment:
/delete-successful-events and /delete-failed-events?

Reply:
Changed